common/persistence/pinot/pinotVisibilityStore_test.go

```diff
@@ -185,7 +185,7 @@ LIMIT 0, 10
 FROM %s
 WHERE DomainID = 'bfd5c907-f899-4baf-a7b2-2ab85e623ebd'
 AND IsDeleted = false
-AND WorkflowID = 'wid' and ((JSON_MATCH(Attr, '"$.CustomStringField" is not null') AND REGEXP_LIKE(JSON_EXTRACT_SCALAR(Attr, '$.CustomStringField', 'string'), 'custom and custom2 or custom3 order by*')) or CustomIntField between 1 and 10)
```

Comment:
previously we didn't find the range query was not working since we used the old pinot table which we had flatten attribute. So the range query "between and" worked.